### PR TITLE
feat(message-system): bumping recommended suite version

### PR DESCRIPTION
**Pull request by MiroslavProchazka**
DO NOT MERGE YET 

Bumping recommended Suite version to 24.1.2, getting rid of some old CJ and Zcash messages

Notion entry in [here](https://www.notion.so/satoshilabs/Old-CJ-messages-cleanup-and-bumping-the-recommended-version-451bccdcb953469c97717ab10dc16ae2?pvs=4)

### CODE REVIEW

**Review by komret — APPROVED** [review]

File: suite-common/message-system/config/config.v1.json [hunk 1]
```diff
@@ -331,7 +260,7 @@
             "conditions": [
                 {
                     "environment": {
-                        "desktop": "<23.9.3",
+                        "desktop": "<24.1.2",
```
Comment (tomasklim) [inline suite-common/message-system/config/config.v1.json]: id of this message has to be changed

Comment (tomasklim) [reply suite-common/message-system/config/config.v1.json]: let's change it to 24.2.. so that all people are on new design

**Review by tomasklim — DISMISSED** [review]

File: suite-common/message-system/config/config.v1.json (hunk 1 above)
Comment (MiroslavProchazka) [reply suite-common/message-system/config/config.v1.json]: Fixed

**Review by MiroslavProchazka — COMMENTED** [review]
Fixed

**Pull request merged**